tasks/selinux_load_module.yml

```diff
@@ -0,0 +1,45 @@
+- name: Set defaults
+  set_fact:
+    name: "{{ item.name | default('')}}"
+    state: "{{ item.state | default('enabled') }}"
+    priority: "{{ item.priority | default('400') }}"
+    store: "{{ selinux_policy | default(ansible_selinux.type, true) }}"
+
+- name: Find installed module
+  set_fact:
+    installed_module: "{{ lookup('selinux_modules', name, priority=priority) | default({ 'name': '', 'priority': '', 'state': 'absent' }, true)}}"
```

Comment:
You could either figure out how to set this in the `vars:` above, or prefix the name with `selinux_`

Reply:
https://github.com/linux-system-roles/selinux/pull/56/commits/3f2f439139571adc5d885e168c64492d982329e4